db/version_builder.cc

```diff
@@ -537,7 +615,23 @@ class VersionBuilder::Rep {
       }
     }
 
-    // Delete files
+    // Add new blob files
```

Comment:
The implementation implicitly requires addition/deletion of blob files to occur before addition of SSTs. Do you think  it's worth documenting?

Reply:
Good point, will add a comment to that effect.